code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -665,34 +665,25 @@
 	id = "borg_upgrade_selfrepair"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/selfrepair
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000)
-	construction_time = 120
+	materials = list(MAT_METAL = 15000, MAT_GLASS = 15000)
+	construction_time = 80
 	category = list("Cyborg Upgrade Modules")
 
 /datum/design/borg_upgrade_expandedsynthesiser
 	name = "Cyborg Upgrade (Hypospray Expanded Synthesiser)"
 	id = "borg_upgrade_expandedsynthesiser"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/hypospray/expanded
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000, MAT_PLASMA=5000)
-	construction_time = 120
-	category = list("Cyborg Upgrade Modules")
-
-/datum/design/borg_upgrade_highstrengthsynthesiser
-	name = "Cyborg Upgrade (Hypospray High-Strength Synthesiser)"
-	id = "borg_upgrade_highstrengthsynthesiser"
-	build_type = MECHFAB
-	build_path = /obj/item/borg/upgrade/hypospray/high_strength
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000, MAT_PLASMA=10000, MAT_URANIUM=5000)
-	construction_time = 120
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 5000, MAT_PLASMA = 5000, MAT_URANIUM = 5000)
+	construction_time = 80
 	category = list("Cyborg Upgrade Modules")
 
 /datum/design/borg_upgrade_piercinghypospray
 	name = "Cyborg Upgrade (Piercing Hypospray)"
 	id = "borg_upgrade_piercinghypospray"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/piercing_hypospray
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000, MAT_TITANIUM=10000, MAT_DIAMOND=5000)
+	materials = list(MAT_METAL = 5000, MAT_GLASS = 5000, MAT_TITANIUM = 5000, MAT_DIAMOND = 4000)
```

Comment:
Flat out a power gaming motive on this by making it cheaper.

Reply:
Agreeing here. Diamond is the rarest material in this list, and it especially shouldn't go from 2+ sheets to exactly 2 sheets.